test/events.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<!--
+Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>iron-label event tests</title>
+    <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+    <script src="../../web-component-tester/browser.js"></script>
+
+    <link rel="import" href="../iron-label.html">
+  </head>
+  <body>
+
+    <iron-label id="inside">
+      TEXT <input type="checkbox">
+    </iron-label>
+
+    <iron-label id="outside" for="outside-checkbox">
+      TEXT
+    </iron-label>
+    <input id="outside-checkbox" type="checkbox">
+
+    <iron-label id="reordered">
+      <span>TEXT</span>
+      <input iron-label-target type="checkbox">
+    </iron-label>
+
+    <script>
+      var inside = document.querySelector('#inside');
+      var outside = document.querySelector('#outside');
+      var reordered = document.querySelector('#reordered');
+
+      suite('event handling', function() {
+
+        suite('target inside', function() {
+          var target = Polymer.dom(inside).firstElementChild;
+
+          test('target has aria-labelledby', function() {
+            var label = target.getAttribute('aria-labelledby');
+            assert.equal(label, 'inside');
+          });
+
+          test('tap on label goes to the target', function() {
+            var e = new CustomEvent('tap');
```

Comment:
Would you consider updating this to use https://github.com/PolymerElements/iron-test-helpers/blob/master/mock-interactions.js ?